### ISSUE TITLE
twisted/application/service.py uses implements

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7160]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Alex reported|
|-|-|
|Trac ID|trac#7160|
|Type|enhancement|
|Created|2014-04-15 20:22:18Z|
|Branch|https://github.com/twisted/twisted/tree/service-implements-7160|

it should use @implementer instead.

<details><summary>Searchable metadata</summary>

```
trac-id__7160 7160
type__enhancement enhancement
reporter__Alex Alex
priority__normal normal
milestone__Python_3_x Python-3.x
branch__branches_service_implements_7160 branches/service-implements-7160
branch_author__hynek hynek
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1397593338000000 1397593338000000
changetime__1397594828000000 1397594828000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|Alex commented|
|-|-|

This is fixed by https://github.com/twisted/twisted/pull/16

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)<a name="note_2"></a>|@hynek commented|
|-|-|

(In [42288]) Branching to service-implements-7160.

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)<a name="note_3"></a>|@hynek commented|
|-|-|

LGTM and the bots agree.

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)<a name="note_4"></a>|@hynek set status to `closed`|
|-|-|

(In [42293]) Merge service-implements-7160

Author: Alex
Reviewer: hynek
Fixes: [#7160](https://github.com/twisted/twisted/issues/7160)
